charts/apisix/values.yaml

```diff
@@ -29,7 +29,7 @@ image:
   pullPolicy: IfNotPresent
   # -- Apache APISIX image tag
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 3.8.0-debian
+  tag: 3.9.0-debian
```

Comment:
we should move `enable_http2`
https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/values.yaml#L274 under `apisix` in this file, just like https://github.com/apache/apisix/pull/11032/files#diff-8d872babc717e9d733641b56bfc530ef98751fbe4e68f08d79b2b83109c22fffR36

Reply:
It seems that there is a better implementation. Please close this PR and look forward to the next release.

Reply:
Is there any progress or plain?